ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -15,7 +15,7 @@
       - "{{ zookeeper.port }}:2181"
 
 - name: wait until the Zookeeper in this host is up and running
-  action: shell echo ruok | nc -w 3 -i 1 {{ inventory_hostname }} {{ zookeeper.port }}
+  action: shell (echo ruok; sleep 1) | nc {{ inventory_hostname }} {{ zookeeper.port }}
```

Comment:
great idea! should remove the bsd vs gnu issue of nc. not all flags are supported everywhere.